libs/components/datetime/src/lib/modules/datepicker/datepicker.component.ts

```diff
@@ -429,4 +437,23 @@ export class SkyDatepickerComponent implements OnDestroy, OnInit {
       this.#customDatesSubscription = undefined;
     }
   }
+
+  #populateInputBox(): void {
+    const hintTextObs: Observable<string | undefined> = this.dateFormat
+      ? this.#resourceSvc.getString(
+          'skyux_datepicker_format_hint_text',
+          this.dateFormat,
+        )
+      : of('');
+
+    hintTextObs.pipe(take(1)).subscribe((formatString) => {
```

Comment:
Using `take` is convenient, but it prevents the subscription from responding to changes to the resources service `getString` method. For example, a new value will be emitted from `getString` when the locale changes.

Reply:
Good call. Updated to use a subject that resets whenever appropriate and completes on destroy.